pkg/config/config_windows.go

```diff
@@ -3,11 +3,8 @@ package config
 import "os"
 
 const (
-	// OverrideContainersConfig holds the default config path overridden by the root user
-	OverrideContainersConfig = "/etc/" + _configPath
-
 	// DefaultContainersConfig holds the default containers config path
-	DefaultContainersConfig = "/usr/share/" + _configPath
+	DefaultContainersConfig = "C:\\ProgramData\\" + _configPath
```

Comment:
_configPath is `containers/containers.conf` so should this use `containers\\containers.conf` instead or does it not matter?
Also given that overrideContainersConfigPath() returns the actual path with the correct env if the defaults was changed should we maybe just leave DefaultContainersConfig empty on windows? At least I see no benifit to read the same file twice other to make it slower.

Reply:
> _configPath is `containers/containers.conf` so should this use `containers\\containers.conf` instead or does it not matter?

It should be fine, the Windows API converts the forward slashes to backslashes. But for consistency I like it more to have `_configPath` const on windows to be `containers\\containers.conf` so I have updated that thanks.

> Also given that overrideContainersConfigPath() returns the actual path with the correct env if the defaults was changed should we maybe just leave DefaultContainersConfig empty on windows? At least I see no benifit to read the same file twice other to make it slower.

Yes I think it's better. I have left it empty.